define/types.go

```diff
@@ -173,11 +173,15 @@ func TempDirForURL(dir, prefix, url string) (name string, subdir string, err err
 	return "", "", errors.New("unreachable code reached")
 }
 
-func cloneToDirectory(url, dir string) ([]byte, string, error) {
+// parseGitBuildContext parses git build context to `repo`, `sub-dir`
+// `branch/commit`, accepts GitBuildContext in the format of
+// `repourl.git#branch/commit:subdir`.
+func parseGitBuildContext(url string) (string, string, string) {
 	gitSubdir := ""
 	gitBranch := ""
 	gitBranchPart := strings.Split(url, "#")
-	var cmd *exec.Cmd
+	// Attempt parsing commit, accepted syntax listed below:
+	// `repourl.git#branch(or commit ID):subdir` or `repourl.git#:subdir` ( uses default branch )
```

Comment:
This doesn't tell me anything that the function's godoc doesn't.  It's redundant.